ServerCore/Pages/Swag/Register.cshtml

```diff
@@ -1,70 +1,70 @@
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="garden-salad" /> Vegan Garden Salad
-                    <div class="food-option-description">Kale, Cucumbers, Tomatoes, Carrots, Edamame, Radishes, Pickled Red Onions and Champagne Vinaigrette. Includes fruit and small dessert.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="turkey-cobb" /> Arugula Turkey Cobb Salad
-                    <div class="food-option-description">Peppery Arugula, Hard-boiled Egg and Creamy Blue Cheese Dressing. Includes fruit and small dessert.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="roast-beef" /> Peppercorn Crusted Roast Beef Sandwich
-                      <div class="food-option-description">House-roasted Beef, Arugula and Horseradish Aioli on Onion Rye. Includes fruit, small dessert and chips.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="roasted-turkey" /> Lemon Pepper Roasted Turkey Sandwich
-                      <div class="food-option-description">Roasted Turkey Breast, Spring Mix and Sliced Tomato on Focaccia. Includes fruit, small dessert and chips.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="smoked-ham" /> Smoked Ham Sandwich
-                      <div class="food-option-description">Smoked Ham, Green Leaf Lettuce and Sliced Tomato on An Herb Torpedo Roll. Includes fruit, small dessert and chips.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="salmon-wrap" /> Blackened Salmon Wrap
-                      <div class="food-option-description">Salmon With Cajun Spice and Baby Kale on A Sundried Tomato Wrap. Includes fruit, small dessert and chips.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="beet-sandwich" /> Vegan Salt-Roasted Beet + Asparagus Sandwich
-                      <div class="food-option-description">Marinated Sliced Beets and Grilled Asparagus on A Potato Roll. Includes fruit, small dessert and chips.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="border-wrap" /> South of the Border Wrap
-                      <div class="food-option-description">Spicy Marinated Chicken, Charred Peppers, Roasted Corn Salsa, Chipotle Aioli in a Sundried Tomato Wrap. Includes chips.</div>
-                <br /><input asp-for="Swag.Lunch" type="radio" class="choice" value="mushroom-wrap" /> Vegan Roasted Mushroom + Red Pepper Wrap
-                      <div class="food-option-description">Rosemary and Balsamic Roasted Portobello, Pickled Peppadew Peppers on a Plain Wrap. Includes chips.</div>
+@page "/{eventId}/{eventRole}/Swag/Register"

```

Comment:
Something messed up the diff with this page. Did you reformat it?

Reply:
Yes, the whitespace was inconsistent and wasn't rendering the code with the proper nesting because of it.  I realize I didn't mention that in the PR comment though, so I'll add that.

Reply:
Do any of the pages you've reformatted also have changes? It's hard to review when both are there and we could use a pointer to the differences

Reply:
Nope, there was only a single field length limit that, per Kenny's recommendation, will be removed in my next/final commit, so once that happens, the only thing done in all the whitespace fixed diffs will be the whitespace fixes.